lib/CreateStaff.dart

```diff
@@ -1,16 +1,935 @@
+                      ]),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    ElevatedButton(
+                      onPressed: () {
+                        {}
+                      },
+                      child: Text('Back'),
+                    ),
+                    ElevatedButton(
+                      onPressed: () {
+                        {}
+                      },
+                      child: Text('Submit'),
+                    ),
+                  ],
+                ),
+              ]),
+            )));
+  }
+
+  Widget createAddressForm(bool createReq) {
+    return (Center(
+        child: Form(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Padding(
+            padding: EdgeInsets.only(bottom: 6.0, left: 60, right: 60, top: 10),
+            child: TextFormField(
+              controller:
+                  createReq ? addressLine1Controller : pAddressLine1Controller,
+              decoration: new InputDecoration(
+                isDense: true,
+                contentPadding:
+                    new EdgeInsets.symmetric(vertical: 10.0, horizontal: 0.0),
+                border: OutlineInputBorder(
+                    borderRadius: BorderRadius.circular(32.0)),
+                focusedBorder: OutlineInputBorder(
+                  borderSide: BorderSide(color: Colors.black, width: 1.0),
+                ),
+                enabledBorder: OutlineInputBorder(
+                  borderSide: BorderSide(color: Colors.black, width: 1.0),
+                ),
+                hintText: 'Addressline 1',
+                labelText: 'Addressline 1',
+              ),
+              validator: (val) {
+                if (val == null || val.isEmpty) {
+                  return 'Please enter Address';
+                }
+              },
+            ),
+          ),
+          Padding(
+            padding: EdgeInsets.only(bottom: 6.0, left: 60, right: 60, top: 10),
+            child: TextFormField(
+              controller:
+                  createReq ? addressLine2Controller : pAddressLine2Controller,
+              decoration: new InputDecoration(
+                isDense: true,
+                contentPadding:
+                    new EdgeInsets.symmetric(vertical: 10.0, horizontal: 10.0),
+                border: OutlineInputBorder(
+                    borderRadius: BorderRadius.circular(32.0)),
+                focusedBorder: OutlineInputBorder(
+                  borderSide: BorderSide(color: Colors.black, width: 1.0),
+                ),
+                enabledBorder: OutlineInputBorder(
+                  borderSide: BorderSide(color: Colors.black, width: 1.0),
+                ),
+                hintText: 'Addressline 2',
+                labelText: 'Addressline 2',
+              ),
+              validator: (val) {
```

Comment:
addressline 2 can be empty, so no need of this validation

Reply:
ok